src/CSV.jl

```diff
@@ -233,6 +234,7 @@ function File(source::Union{String, IO},
     missingstrings,
     missingstring,
     delim::Union{Char, String},
+    ignore_repeated_delimiters::Bool,
```

Comment:
Can we use a shorter name, ideally without underscores? I cry everytime I write `rows_for_type_detect`...

Reply:
Ugly names for ugly features; that's what I say....

Reply:
https://github.com/JuliaData/CSV.jl/pull/266